blocks/library/button/index.js

```diff
@@ -75,19 +75,9 @@ registerBlockType( 'core/button', {
 					onChange={ ( value ) => setAttributes( { text: value } ) }
 					formattingControls={ [ 'bold', 'italic', 'strikethrough' ] }
 					keepPlaceholderOnFocus
+					extraToolbarButtons={ <UrlInputButton showSettings={ false } url={ url } onChange={ updateUrl } /> }
```

Comment:
Why do we need this prop? Can't we just put this putting in the `BlockControls` component?